src/apps/applets/domain/applet.py

```diff
@@ -136,6 +137,19 @@ class AppletActivitiesDetailsPublic(PublicModel):
     respondent_meta: dict | None = None
 
 
+class ActivityLanguageWithItemsMobileDetailPublicType(ActivityLanguageWithItemsMobileDetailPublic):
+    type = "activity"
+
+
+class PublicFlowFullType(PublicFlowFull):
+    type = "flow"
```

Comment:
Nit: suggestion to avoid ambiguity with other kinds of flows (item flows)? Also NBD
```suggestion
    type = "activityFlow"
```